testing.md

```diff
@@ -0,0 +1,79 @@
+# Testing
+
+## Best Practices
+
+Writing good tests in javascript is similar to writing good tests in any other language. There are a few best practices that are more pertinent to javascript testing.
+
+### Mocking
+
+Mock any dependency with `sinon.js`. `sinon.js` is extremely easy to use.
+TODO
+
+### No AJAX calls
+TODO
```

Comment:
well, since we have a lot of ajax on our pages, and presumably we'll move to even more...it would be a good idea to include mocking ajax requests in a future update. :) @orangejenny @benrudolph